package.json

```diff
@@ -129,6 +129,9 @@
     "semver": "^7.3.7",
     "vscode-nls": "^5.0.0"
   },
+  "extensionDependencies": [
+    "vscode.npm"
```

Comment:
Will this cause any issues for people that are using yarn, etc?

Reply:
As far as I know, package managers just ignore properties in package.json that they don't know about. [This](https://stackoverflow.com/a/27232456/3507557) StackOverflow answer also seems to indicate that this is generally valid. `extensionDependencies` is part of VSCode's extension manifest specification: https://code.visualstudio.com/api/references/extension-manifest